demo/index.md

```diff
@@ -27,28 +27,28 @@ See <auro-hyperlink target="_blank" href="https://webcode.tools/generators/css/b
 <div class="exampleWrapper">
   <!-- AURO-GENERATED-CONTENT:START (FILE:src=./../../apiExamples/basic.html) -->
   <!-- The below content is automatically added from ./../../apiExamples/basic.html -->
-  <auro-background bg="url(https://sitecore-test-single-westus2.azurewebsites.net/-/media/Images/pages/examples/ad2) center center/cover no-repeat">
```

Comment:
Need to update generateDocs script and delete all references to `apiExamples.md`